java/src/org/openqa/selenium/chromium/ChromiumDriver.java

```diff
@@ -41,13 +41,15 @@
 import org.openqa.selenium.bidi.BiDi;
 import org.openqa.selenium.bidi.BiDiException;
 import org.openqa.selenium.bidi.HasBiDi;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.devtools.CdpEndpointFinder;
 import org.openqa.selenium.devtools.CdpInfo;
 import org.openqa.selenium.devtools.CdpVersionFinder;
 import org.openqa.selenium.devtools.Connection;
 import org.openqa.selenium.devtools.DevTools;
 import org.openqa.selenium.devtools.HasDevTools;
 import org.openqa.selenium.devtools.noop.NoOpCdpInfo;
+import org.openqa.selenium.edge.EdgeOptions;
```

Comment:
I understand your motivation for this change. But this is might not best approach for it. Ideally, the EdgeDriver and ChromeDriver extend the ChromiumDriver. The superclass should not be dependent on the details of the implementing subclasses.

Reply:
@pujagani Any suggestions to avoid repeated code, or should we leave it as it is for now?

Reply:
When I refactored the Services code to make Edge & Chrome match I started to look at abstracting out a superclass, and it didn't look simple, so I backed away slowly. It would be good to figure it out, but it's not a priority.

Reply:
Well if we are all on the same page, we can close this PR for now without the merge.